snapshotter/devmapper/snapshotter.go

```diff
@@ -186,11 +206,46 @@ func (s *Snapshotter) Commit(ctx context.Context, name, key string, opts ...snap
 	log.G(ctx).WithFields(logrus.Fields{"name": name, "key": key}).Debug("commit")
 
 	return s.withTransaction(ctx, true, func(ctx context.Context) error {
-		_, err := storage.CommitActive(ctx, key, name, snapshots.Usage{}, opts...)
+		id, info, _, err := storage.GetInfo(ctx, key)
```

Comment:
And why is this usage not important? Is it assumed that the snapshot will always be active at this point?

Reply:
Because we don't keep usage info for active snapshots in database.